packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -99,9 +107,10 @@ function createIframeVirtualEnvironment(
         blueCreateHooksCallbackCache.set(blueRefs.document, blueConnector);
     }
     const { eval: redIndirectEval } = redWindow;
+    const sourceTextCallback = (v: string) => redIndirectEval(trustedCreateScript(v));
     const env = new VirtualEnvironment({
```

Comment:
I think the rename request was for the option name. But I think that we'll end up needing 2 callbacks eventually. One for signing and one for transforming.